WolvenKit.App/Controllers/RED4Controller.cs

```diff
@@ -242,10 +245,39 @@ public async Task<bool> PackAndInstallProject()
 
             InstallMod();
 
+            await DeployRedmod();
+
             _progressService.IsIndeterminate = false;
             return true;
         }
 
+        public async Task<bool> DeployRedmod()
+        {
+            if (!_pluginService.IsInstalled(EPlugin.redmod))
+            {
+                return false;
+            }
+
+            var result = false;
+            // compile with redmod
+            var redmodPath = Path.Combine(_settingsManager.GetRED4GameRootDir(), "tools", "redmod", "bin", "redmod.exe");
+            if (File.Exists(redmodPath))
+            {
+                var rttiSchemaPath = Path.Combine(_settingsManager.GetRED4GameRootDir(), "tools", "redmod", "metadata.json");
+                var args = $"deploy -root=\"{_settingsManager.GetRED4GameRootDir()}\" -rttiSchemaPath=\"{rttiSchemaPath}\"";
+
+                _loggerService.Info($"WorkDir: {redmodPath}");
+                _loggerService.Info($"Running commandlet: {args}");
+                result = await ProcessUtil.RunProcessAsync(redmodPath, args);
+            }
+            else
+            {
+                result = await Task.FromResult(true);
+            }
+
+            return result;
```

Comment:
Here I would just do `return Task.FromResult(true)` instead of this if-else check.

In the if, you can also do a direct return, you don't even need to await imo. Just start the task the the callee will await for it, no need to wait and then return the result.